bundles/AdminBundle/Resources/public/js/pimcore/object/tags/video.js

```diff
@@ -211,9 +211,11 @@ pimcore.object.tags.video = Class.create(pimcore.object.tags.abstract, {
     },
 
     empty: function () {
+        let allowedTypes = this.data.allowedTypes;
```

Comment:
this doesn't work because of this after confirming with `openEdit`
https://github.com/pimcore/pimcore/blob/a0dc084959c787a1d05ed942a5dc7b623f3be6cd/bundles/AdminBundle/Resources/public/js/pimcore/object/tags/video.js#L170

but we could use this
```suggestion
        let allowedTypes = this.fieldConfig.allowedTypes;
```